packages/core/http/core-http-server/src/router/route.ts

```diff
@@ -207,6 +207,14 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
    * @remarks This will be surfaced in OAS documentation.
    */
   deprecated?: boolean;
+
+  /**
+   * Release version or date that this route will be removed
+   * Use with `deprecated: true`
+   *
+   * @remarks This will be surfaced in OAS documentation.
```

Comment:
What do you think about adding an example to this doc comment?

```suggestion
   * @remarks This will be surfaced in OAS documentation.
   * @example 9.0.0
```